sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -1025,4 +1025,9 @@ public interface OptionsConfiguration<T extends SentryOptions> {
   public static @NotNull SentryId captureCheckIn(final @NotNull CheckIn checkIn) {
     return getCurrentHub().captureCheckIn(checkIn);
   }
+
+  public static void captureReplay(
+      final @NotNull SentryReplayEvent replay, final @Nullable Hint hint) {
+    getCurrentHub().captureReplay(replay, hint);
+  }
```

Comment:
m: Why is this public? In what scenario can the user manually create a replay event?

IMO we should not expose any replay related object in the first versions.

Reply:
yeah, I wanted to remove it but overlooked 👍 thanks